src/iterative_sorting/iterative_sorting.py

```diff
@@ -3,23 +3,23 @@ def selection_sort( arr ):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
-        smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(i + 1, len(arr)):
+            if arr[cur_index] > arr[j]:
+                cur_index = j
         # TO-DO: swap
-
-
-
+        arr[i], arr[cur_index] = arr[cur_index], arr[i]
 
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
+    for i in range(len(arr)):
```

Comment:
Clean and standard. I know this is a straight forward bubble sort. Add comments to showcase your understanding of it.